(Tue Nov 19 16:02:06 2019)(-0.130409 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/farmer/anaconda3/envs/general/lib/python3.6/site-packages/pyscannerbit/ScannerBit/lib/libscanner_diver_1.0.0.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-0ai_8_ii/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(-0.130389 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/farmer/anaconda3/envs/general/lib/python3.6/site-packages/pyscannerbit/ScannerBit/lib/libscanner_diver_1.0.2.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-0ai_8_ii/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(-0.11958 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/farmer/anaconda3/envs/general/lib/python3.6/site-packages/pyscannerbit/ScannerBit/lib/libscanner_great.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-0ai_8_ii/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(-0.106133 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/farmer/anaconda3/envs/general/lib/python3.6/site-packages/pyscannerbit/ScannerBit/lib/libscanner_multinest_3.10.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-0ai_8_ii/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0194438 [s])[Default,Utilities]:
Random number engine mt19937_64 (default) selected with seed -1300657656.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0201803 [s])[Default,Printers][Info]:
Running shell command: rm -f pyscannerbit_run_data/random_scan/samples//results.hdf5
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0303639 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Plugin name is not defined under the "scanners" tag.  using the tag "badass" as the plugin name.

Raised at: line 625 in function void Gambit::Scanner::Plugins::pluginInfo::iniFile(const Gambit::Options&) of /tmp/pip-req-build-0ai_8_ii/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0306171 [s])[Default,Core]:
Starting scan.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.031873 [s])[Default,Printers][Info]:
Assigned printer ID '0' to output quantity with label 'LogLike'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0319369 [s])[Default,Printers][Info]:
Assigned printer ID '1' to output quantity with label 'pointID'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0319861 [s])[Default,Printers][Info]:
Assigned printer ID '2' to output quantity with label 'MPIrank'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0326801 [s])[Default,Printers][Info]:
Assigned printer ID '3' to output quantity with label 'default::z'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0327951 [s])[Default,Printers][Info]:
Assigned printer ID '4' to output quantity with label 'default::x'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0328712 [s])[Default,Printers][Info]:
Assigned printer ID '5' to output quantity with label 'default::y'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:06 2019)(0.0332429 [s])[Default,Printers][Info]:
Assigned printer ID '6' to output quantity with label 'unitCubeParameters'
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:08 2019)(2.08878 [s])[Default]:
Synchronised buffers flushed for rank 0 printers. Waiting for all processes to flush their sync data before we try to write the RA data.
--<>--<>--<>--<>--<>--<>--<>--
(Tue Nov 19 16:02:08 2019)(2.08922 [s])[Default]:
GAMBIT run completed successfully.
--<>--<>--<>--<>--<>--<>--<>--
